app/pb_kits/playbook/pb_textarea/_textarea.html.erb

```diff
@@ -0,0 +1,19 @@
+<%= content_tag(:div,
+    id: object.id,
+    data: object.data,
+    class: object.classname) do %>
+  <% if object.label.present? %>
+    <%= pb_rails("caption", props: {text: object.label, dark: object.dark}) %>
+  <% end %>
+  <% if object.children %>
+    <%= capture(&object.children) %>
+  <% else %>
+    <%= text_area(
+        nil, 
+        nil, 
```

Comment:
@thestephenmarshall Thoughts?

Reply:
These need to be props [as outlined here](https://apidock.com/rails/ActionView/Helpers/FormHelper/text_area) `text_area(object_name, method...` are the arguments needed to properly build the textarea attributes `name` and `id` which are required in Rails forms. I haven't tried with strings before, but they should work